### PR TITLE
Add POST and PUT examples in mutators API doc

### PR DESCRIPTION
**Pull request by hillaryfraley**
Add POST /mutators and PUT /mutators/:mutator examples.

Epic #1452

### CODE REVIEW

File: content/sensu-go/5.16/api/mutators.md [hunk 1]
```diff
@@ -27,7 +27,9 @@ The `/mutators` API endpoint provides HTTP GET access to [mutator][1] data.
 The following example demonstrates a request to the `/mutators` API endpoint, resulting in a JSON array that contains [mutator definitions][1].
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/core/v2/namespaces/default/mutators -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET
```
Comment (Blue0ctober) [inline content/sensu-go/5.16/api/mutators.md]: requires ' \ ' at the end of the line `curl -X GET \`

File: content/sensu-go/5.16/api/mutators.md [hunk 2]
```diff
@@ -27,7 +27,9 @@ The `/mutators` API endpoint provides HTTP GET access to [mutator][1] data.
...
+http://127.0.0.1:8080/api/core/v2/namespaces/default/mutators
```
Comment (Blue0ctober) [inline content/sensu-go/5.16/api/mutators.md]: Same as above `http://127.0.0.1:8080/api/core/v2/namespaces/default/mutators \`

File: content/sensu-go/5.16/api/mutators.md [hunk 3]
```diff
@@ -107,7 +135,8 @@ The `/mutators/:mutator` API endpoint provides HTTP GET access to [mutator data]
 In the following example, querying the `/mutators/:mutator` API endpoint returns a JSON map that contains the requested [`:mutator` definition][1] (in this example, for the `:mutator` named `example-mutator`).
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/core/v2/namespaces/default/mutators/example-mutator -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET http://127.0.0.1:8080/api/core/v2/namespaces/default/mutators/example-mutator
```
Comment (Blue0ctober) [inline content/sensu-go/5.16/api/mutators.md]: requires ' \ ' at the end of the line for multi line command. 'curl -X GET' should probably be on its own line like the previous example.

**Review by Blue0ctober — CHANGES_REQUESTED** [review]
The curl commands will not work as-is, see comments for fixes.

File: content/sensu-go/5.16/api/mutators.md (hunk 1 above)
Comment (hillaryfraley) [inline content/sensu-go/5.16/api/mutators.md]:
```suggestion
curl -X GET \
```

File: content/sensu-go/5.16/api/mutators.md (hunk 2 above)
Comment (hillaryfraley) [inline content/sensu-go/5.16/api/mutators.md]:
```suggestion
http://127.0.0.1:8080/api/core/v2/namespaces/default/mutators \
```

File: content/sensu-go/5.16/api/mutators.md (hunk 3 above)
Comment (hillaryfraley) [inline content/sensu-go/5.16/api/mutators.md]:
```suggestion
curl -X GET \
http://127.0.0.1:8080/api/core/v2/namespaces/default/mutators/example-mutator \
```

**Pull request merged**